### ISSUE TITLE
[FR] - Event Streaming

### ISSUE THREAD

**Issue by nikotili** [input-output-hk/cardano-node#3227]
**External**

**Network Event Streaming**
A network event streaming service might be needed to allow developers to implement subscriptions for certain events like `Pending Transactions`, `New Block Creations`, `Logging`, etc. 

In my case, I'm implementing a listener for incoming transactions to a certain address and I have found this examples:
- https://developers.cardano.org/docs/integrate-cardano/listening-for-payments-wallet
- https://developers.cardano.org/docs/integrate-cardano/listening-for-payments-cli

They seem to work fine for simple payment cases, but still, support of event streaming by nodes would be a lot more helpful and scalable.

**Comment by johnnynuuma** [comment]
I agree.  Would like to see Cardano implement a generic way to log and listen for events ( both poll and push models ) a la Ethereum.

**Comment by Jimbo4350** [comment]
Closing this. If this is still relevant please reopen.

**Comment by johnnynuuma** [comment]
Please reopen

**Comment by ltouro** [comment]
Check this: https://cardanosolutions.github.io/kupo and (more broadly) this: https://www.essentialcardano.io/article/a-list-of-community-built-developer-tools-on-cardano. 

If you really looking for a (low level) node-based solution, check this: https://ogmios.dev/mini-protocols/

**Comment by Jimbo4350** [comment]
@kosyrevSerge does the new logging framework provide anything like this?
